EventManager/host/build.gradle

```diff
@@ -18,11 +18,7 @@ dependencies {
     implementation project(":common")
     implementation "com.r3.conclave:conclave-host:$conclaveVersion"
     implementation 'org.springframework.boot:spring-boot-starter-web'
-    if (mode == "mock") {
-        implementation project(path: ":enclave")
-    } else {
-        implementation project(path: ":enclave", configuration: mode)
-    }
+    implementation project(path: ":enclave")
```

Comment:
Correct me if I am wrong, but I am not sure this is enough.
If I run this now
` ./gradlew host:run -PenclaveMode=simulation --info `
I can see that we are not building in simulation mode.
The default of this file is a mock mode (see the code above), which is overriding the default simulation mode.
I think we need to just leave this 
` implementation project(path: ":enclave", configuration: mode) `
(removing the branching)
and pass the (manual) full string of the Enclave in EventManagerHost
I.e. replace this
`   enclave = EnclaveHost.load(EventManagerEnclave::class.java.canonicalName);                                                                                                                                                   `
with 
`        enclave = EnclaveHost.load("me.gendal.conclave.eventmanager.enclave.EventManagerEnclave");                                                                                                                                                     
`